ownCloud/Client/Viewer/PDF/PDFViewerViewController.swift

```diff
@@ -7,362 +7,364 @@
-            constraints.append(thumbnailView.heightAnchor.constraint(equalTo: self.view.heightAnchor, multiplier: thumbnailViewHeightMultiplier))
-
-            constraints.append(containerView.leadingAnchor.constraint(equalTo: guide.leadingAnchor))
-            constraints.append(containerView.trailingAnchor.constraint(equalTo: guide.trailingAnchor))
-            constraints.append(containerView.bottomAnchor.constraint(equalTo: thumbnailView.topAnchor))
-
-        case .none:
-            constraints.append(containerView.leadingAnchor.constraint(equalTo: guide.leadingAnchor))
-            constraints.append(containerView.trailingAnchor.constraint(equalTo: guide.trailingAnchor))
-            constraints.append(containerView.bottomAnchor.constraint(equalTo: guide.bottomAnchor))
-            thumbnailView.isHidden = true
-        }
-
-        self.activeViewConstraints = constraints
-
-    }
-
-    fileprivate func setupToolbar() {
-        searchButtonItem = UIBarButtonItem(barButtonSystemItem: .search, target: self, action: #selector(search))
-        gotoButtonItem = UIBarButtonItem(image: UIImage(named: "ic_pdf_go_to_page"), style: .plain, target: self, action: #selector(goToPage))
-        outlineItem = UIBarButtonItem(image: UIImage(named: "ic_pdf_outline"), style: .plain, target: self, action: #selector(showOutline))
-
-        self.parent?.navigationItem.rightBarButtonItems = [
-            gotoButtonItem!,
-            searchButtonItem!,
-            outlineItem!]
-    }
-
-    fileprivate func selectPage(with label:String) {
-        guard let pdf = pdfView.document else { return }
-
-        if let pageNr = Int(label) {
-            if pageNr > 0 && pageNr < pdf.pageCount {
-                if let page = pdf.page(at: pageNr - 1) {
-                    self.pdfView.go(to: page)
-                }
-            } else {
-                let alertController = UIAlertController(title: "Invalid Page".localized,
-                                                        message: "The entered page number doesn't exist".localized,
-                                                        preferredStyle: .alert)
-                alertController.addAction(UIAlertAction(title: "OK".localized, style: .default, handler: nil))
-                self.present(alertController, animated: true, completion: nil)
-            }
-        }
-    }
-
-    fileprivate func updatePageLabel() {
-        guard let pdf = pdfView.document else { return }
-
-        guard let page = pdfView.currentPage else { return }
-
-        let pageNrText = "\(pdf.index(for: page) + 1)"
-        let maxPageCountText = "\(pdf.pageCount)"
-        pageCountLabel.text = NSString(format: "%@ of %@".localized as NSString, pageNrText, maxPageCountText) as String
-    }
+		self.activeViewConstraints = constraints
+
+	}
+
+	fileprivate func setupToolbar() {
+		searchButtonItem = UIBarButtonItem(barButtonSystemItem: .search, target: self, action: #selector(search))
```

Comment:
would be cool, to add accessibility labels to all bar button items

Reply:
@hosy added accessibility labels